docs/source/tutorials/getting_started/dependencies.rst

```diff
@@ -43,3 +43,14 @@ Full list of dependencies can be found at install_macos_dependencies_build_.
   :code: shell
 
 .. _install_ubuntu_dependencies_build: ../../../../../.github/workflows/install_macos_dependencies_build.sh
+
```

Comment:
This should be on the main README page, under build instructions:
https://github.com/os-fpga/FOEDAG/edit/main/README.md